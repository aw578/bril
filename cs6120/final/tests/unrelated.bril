@main {
  x1: int = const 1;
  x2: int = const 2;
  t1: int = add x1 x2;
  t3: int = add x1 t1;
  t4: int = add t1 x2;
}