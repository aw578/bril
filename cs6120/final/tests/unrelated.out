Congruence classes for function main:
Class const 1: ['x1.0']
Class const 2: ['x2.0']
Class add: ['t1.0']
Class add_new: ['t3.0']
Class add_new_new: ['t4.0']
